tests/test_packages/test_skills/test_weather.py

```diff
@@ -20,251 +20,59 @@
-                "config",
-                "set",
-                "agent.default_connection",
-                "fetchai/oef:0.2.0",
-            ],
-            standalone_mode=False,
-        )
-        assert result.exit_code == 0
-
-        result = self.runner.invoke(
-            cli,
-            [
-                *CLI_LOG_OPTION,
-                "add",
-                "--local",
-                "skill",
-                "fetchai/weather_station:0.1.0",
-            ],
-            standalone_mode=False,
-        )
-        assert result.exit_code == 0
-
-        # Load the agent yaml file and manually insert the things we need
-        yaml_path = os.path.join(
-            "vendor", "fetchai", "skills", "weather_station", "skill.yaml"
-        )
-        file = open(yaml_path, mode="r")
-
-        # read all lines at once
-        whole_file = file.read()
-
-        whole_file = whole_file.replace("is_ledger_tx: true", "is_ledger_tx: false")
-
-        # close the file
-        file.close()
-
-        with open(yaml_path, "w") as f:
-            f.write(whole_file)
-
-        result = self.runner.invoke(
-            cli, [*CLI_LOG_OPTION, "install"], standalone_mode=False
-        )
-        assert result.exit_code == 0
-
-        os.chdir(self.t)
+        self.add_item("connection", "fetchai/oef:0.1.0")
+        self.add_item("skill", "fetchai/weather_station:0.1.0")
+        self.disable_ledger_tx("fetchai", "skill", "weather_station")
+        self.run_install()
 
-        # add packages for agent two and run it
-        agent_two_dir_path = os.path.join(self.t, self.agent_name_two)
+        # prepare agent two (weather client)
+        agent_two_dir_path = os.path.join(self.t, agent_name_two)
         os.chdir(agent_two_dir_path)
 
-        result = self.runner.invoke(
-            cli,
-            [*CLI_LOG_OPTION, "add", "--local", "connection", "fetchai/oef:0.2.0"],
-            standalone_mode=False,
-        )
-        assert result.exit_code == 0
+        self.add_item("connection", "fetchai/oef:0.1.0")
+        self.add_item("skill", "fetchai/weather_client:0.1.0")
+        self.disable_ledger_tx("fetchai", "skill", "weather_client")
+        self.run_install()
 
-        result = self.runner.invoke(
-            cli,
-            [
-                *CLI_LOG_OPTION,
-                "config",
-                "set",
-                "agent.default_connection",
-                "fetchai/oef:0.2.0",
-            ],
-            standalone_mode=False,
-        )
-        assert result.exit_code == 0
-
-        result = self.runner.invoke(
-            cli,
-            [
-                *CLI_LOG_OPTION,
-                "add",
-                "--local",
-                "skill",
-                "fetchai/weather_client:0.1.0",
-            ],
-            standalone_mode=False,
-        )
-        assert result.exit_code == 0
-
-        # Load the agent yaml file and manually insert the things we need
```

Comment:
you're missing this..

Reply:
It's here.
https://github.com/fetchai/agents-aea/pull/1098/files#diff-a9f80421196c9acba163cbf08ef4947fR60

Reply:
Cannot see it there.

Reply:
Line 60 this file. Don't know why Github leads to other place.
https://github.com/fetchai/agents-aea/pull/1098/files/039c2e6064a9f9195b812e8bd37cce12281343fa#diff-a9f80421196c9acba163cbf08ef4947fR60